pallets/asset-manager/src/weights.rs

```diff
@@ -101,6 +103,12 @@ impl<T: frame_system::Config> WeightInfo for SubstrateWeight<T> {
 			.saturating_add(T::DbWeight::get().reads(3 as Weight))
 			.saturating_add(T::DbWeight::get().writes(6 as Weight))
 	}
+	#[rustfmt::skip]
```

Comment:
Are these benchmark results "official?"

Reply:
Also, they don't have the individual storage item comments like the others have. Those are awesome for auditing the results

Reply:
Yeah they are official, I just run the one from register_local_asset but probably forgot adding the comments. Will do that